src/mailadm/util.py

```diff
@@ -21,7 +21,11 @@ def parse_expiry_code(code):
         raise ValueError("expiry codes are at least 2 characters")
     val = int(code[:-1])
     c = code[-1]
-    if c == "w":
```

Comment:
there could be some quick unit tests for "parse_expiry_code"

Reply:
There are already tests, I added months and years :)